webrender/src/render_backend.rs

```diff
@@ -261,7 +265,10 @@ impl RenderBackend {
                         }
                         ApiMsg::RequestWebGLContext(size, attributes, tx) => {
                             if let Some(ref wrapper) = self.webrender_context_handle {
-                                let result = wrapper.new_context(size, attributes);
+                                let dispatcher = Box::new(WebRenderGLDispatcher {
```

Comment:
Yeah, I'll add a macro. It will avoid unneeded allocations in the platforms that don't use this.